src/rules/noUnusedVariableRule.ts

```diff
@@ -30,6 +30,10 @@ export class Rule extends Lint.Rules.TypedRule {
         description: Lint.Utils.dedent`Disallows unused imports, variables, functions and
             private class members. Similar to tsc's --noUnusedParameters and --noUnusedLocals
             options, but does not interrupt code compilation.`,
+        descriptionDetails: Lint.Utils.dedent`If addition to avoiding compilation errors,
```

Comment:
"In addition ..."

Reply:
Also you can add a newline + indentation before "If"

Reply:
Fixed `if`, and moved the start of the sentence to a newline.